File: application/src/main/java/com/study/platform/domain/team/usecase/UpdateTeamScheduleUseCase.java
package com.study.platform.domain.team.usecase;

import com.study.platform.domain.team.dto.request.TeamScheduleUpdateRequest;
import com.study.platform.domain.team.dto.response.TeamScheduleResponse;

import java.util.UUID;

public interface UpdateTeamScheduleUseCase {
    TeamScheduleResponse execute(UUID userId, UUID teamId, UUID scheduleId, TeamScheduleUpdateRequest request);
}

File: application/src/main/kotlin/com/study/platform/domain/team/usecase/UpdateTeamScheduleUseCase.kt
package com.study.platform.domain.team.usecase

import com.study.platform.domain.team.dto.request.TeamScheduleUpdateRequest
import com.study.platform.domain.team.dto.response.TeamScheduleResponse
import java.util.UUID

interface UpdateTeamScheduleUseCase {
    fun execute(userId: UUID, teamId: UUID, scheduleId: UUID, request: TeamScheduleUpdateRequest): TeamScheduleResponse
}
